package.json

```diff
@@ -241,9 +241,8 @@
     "@expo/config-plugins/glob": "^10.3.10",
     "@solana/web3.js/rpc-websockets": "^8.0.1",
     "@metamask/nonce-tracker@npm:^5.0.0": "patch:@metamask/nonce-tracker@npm%3A5.0.0#~/.yarn/patches/@metamask-nonce-tracker-npm-5.0.0-d81478218e.patch",
-    "@metamask/network-controller@npm:^17.0.0": "patch:@metamask/network-controller@npm%3A21.0.0#~/.yarn/patches/@metamask-network-controller-npm-21.0.0-559aa8e395.patch",
-    "@metamask/network-controller@npm:^19.0.0": "patch:@metamask/network-controller@npm%3A21.0.0#~/.yarn/patches/@metamask-network-controller-npm-21.0.0-559aa8e395.patch",
-    "@metamask/network-controller@npm:^20.0.0": "patch:@metamask/network-controller@npm%3A21.0.0#~/.yarn/patches/@metamask-network-controller-npm-21.0.0-559aa8e395.patch",
+    "@metamask/network-controller@npm:^19.0.0": "patch:@metamask/network-controller@npm%3A22.0.2#~/.yarn/patches/@metamask-network-controller-npm-22.0.2-6fc65c500a.patch",
```

Comment:
Looks like we can remove this entry as well. I can't see a reference to it in the lockfile

Reply:
```suggestion
```

Reply:
Yes my bad I was looking at lock before running yarn install